_component_design/status-labels.md

```diff
@@ -0,0 +1,150 @@
+---
+
+{% include toc.html %}
+
+## Introduction
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+Use status labels to communicate the current state of an object or process. For example, to communicate the status of a support ticket, or current health of a device.
+
+### Best Practices
+
+- To reduce visual noise when using in a table or list, label width should be fixed at the width that best accomodates the longest label with a minimum padding of 8px. 
+- Adhere to a recommended max character limit of 20 characters.
+- Use color and stroke vs fill to control how much attention a given status label commands.
+- Do not use other colors from the Helix color palette.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Status label within the CTUI project"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/notifications/status-labels/status-labels-hero.png){:width="1440px"}
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+</section>
+
+
+## Specifications
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Default layout
+
+Status labels are wide tags that include short, brief, and helpful text communication regarding the progress of our users business.
+
+Status labels differentiate themselves from buttons and links by utilizing 100% rounded border radius, in addition to, neutral or alert color strategy which differs from our default buttons and links.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"A default status label specification"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/notifications/status-labels/status-labels-default.png){:width="499px"}
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Emphasis
+
+The status label is used to communicate the status of an object or process. You can emphasize or de-emphasize that status data within a UI by modifying the stroke and fill or color property on a status label.
+
+#### Stroke vs fill
+
+- Applying stroke only to a status label de-emphasizes the content while using fill emphasizes the content. 
+- Use all-stroke only status labels when the information contained within is not the most important information on the screen.
+- Conversely, use an all-filled status label variation when the information is of high importance on the screen.
+- If you are using an all-stroke system, you may use fill for a status if you want to draw attention to that status in particular.
```

Comment:
Does all of this conflict with the best practices? I'm a bit confused at this point.

Reply:
These are when to use 'Stroke versus fill'. I will agree it is a little confusing, maybe the author @palindromebart can weigh in.